<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormWallpaper</class>
 <widget class="QWidget" name="FormWallpaper">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>490</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>490</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>981</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frameWallpaper">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>微软雅黑</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>360</y>
        <width>401</width>
        <height>121</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="spacing">
        <number>3</number>
       </property>
       <item row="1" column="3">
        <widget class="xIndicatorButton" name="BtnY02">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">吸具1-E1Y03</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>40</height>
          </size>
         </property>
         <property name="iconOn" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOn.png</normaloff>:/img/widget/buttonOn.png</iconset>
         </property>
         <property name="iconOff" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOff.png</normaloff>:/img/widget/buttonOff.png</iconset>
         </property>
         <property name="contentAlignLeft" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="xIndicatorButton" name="BtnY03">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">吸具2-E1Y04</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>40</height>
          </size>
         </property>
         <property name="iconOn" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOn.png</normaloff>:/img/widget/buttonOn.png</iconset>
         </property>
         <property name="iconOff" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOff.png</normaloff>:/img/widget/buttonOff.png</iconset>
         </property>
         <property name="contentAlignLeft" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="xIndicatorButton" name="BtnY04">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">抱具1-E1Y05</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>40</height>
          </size>
         </property>
         <property name="iconOn" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOn.png</normaloff>:/img/widget/buttonOn.png</iconset>
         </property>
         <property name="iconOff" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOff.png</normaloff>:/img/widget/buttonOff.png</iconset>
         </property>
         <property name="contentAlignLeft" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="xIndicatorButton" name="BtnY05">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">抱具2-E1Y06</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>40</height>
          </size>
         </property>
         <property name="iconOn" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOn.png</normaloff>:/img/widget/buttonOn.png</iconset>
         </property>
         <property name="iconOff" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOff.png</normaloff>:/img/widget/buttonOff.png</iconset>
         </property>
         <property name="contentAlignLeft" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="xIndicator" name="LedX03">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="iconOn">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedGreenOn.png</normaloff>:/img/widget/LedGreenOn.png</iconset>
         </property>
         <property name="iconOff">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedOff.png</normaloff>:/img/widget/LedOff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">吸具2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="xIndicatorButton" name="BtnY06">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">抱具3-E1Y07</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>40</height>
          </size>
         </property>
         <property name="iconOn" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOn.png</normaloff>:/img/widget/buttonOn.png</iconset>
         </property>
         <property name="iconOff" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOff.png</normaloff>:/img/widget/buttonOff.png</iconset>
         </property>
         <property name="contentAlignLeft" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="xIndicatorButton" name="BtnY07">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string notr="true">抱具4-E1Y08</string>
         </property>
         <property name="iconSize">
          <size>
           <width>14</width>
           <height>40</height>
          </size>
         </property>
         <property name="iconOn" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOn.png</normaloff>:/img/widget/buttonOn.png</iconset>
         </property>
         <property name="iconOff" stdset="0">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/buttonOff.png</normaloff>:/img/widget/buttonOff.png</iconset>
         </property>
         <property name="contentAlignLeft" stdset="0">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="xIndicator" name="LedX04">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="iconOn">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedGreenOn.png</normaloff>:/img/widget/LedGreenOn.png</iconset>
         </property>
         <property name="iconOff">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedOff.png</normaloff>:/img/widget/LedOff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">抱具1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="xIndicator" name="LedX06">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="iconOn">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedGreenOn.png</normaloff>:/img/widget/LedGreenOn.png</iconset>
         </property>
         <property name="iconOff">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedOff.png</normaloff>:/img/widget/LedOff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">抱具3</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="xIndicator" name="LedX05">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="iconOn">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedGreenOn.png</normaloff>:/img/widget/LedGreenOn.png</iconset>
         </property>
         <property name="iconOff">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedOff.png</normaloff>:/img/widget/LedOff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">抱具2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="xIndicator" name="LedX07">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="iconOn">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedGreenOn.png</normaloff>:/img/widget/LedGreenOn.png</iconset>
         </property>
         <property name="iconOff">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedOff.png</normaloff>:/img/widget/LedOff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">抱具4</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="xIndicator" name="LedX02">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="iconOn">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedGreenOn.png</normaloff>:/img/widget/LedGreenOn.png</iconset>
         </property>
         <property name="iconOff">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedOff.png</normaloff>:/img/widget/LedOff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">吸具1</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="xIndicator" name="LedX01">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="iconOn">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedGreenOn.png</normaloff>:/img/widget/LedGreenOn.png</iconset>
         </property>
         <property name="iconOff">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedOff.png</normaloff>:/img/widget/LedOff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">垂直到位</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="xIndicator" name="LedX00">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>250</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>微软雅黑</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="iconOn">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedGreenOn.png</normaloff>:/img/widget/LedGreenOn.png</iconset>
         </property>
         <property name="iconOff">
          <iconset resource="../xPad.qrc">
           <normaloff>:/img/widget/LedOff.png</normaloff>:/img/widget/LedOff.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="text">
          <string notr="true">水平到位</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xIndicator</class>
   <extends>QWidget</extends>
   <header>xindicator.h</header>
  </customwidget>
  <customwidget>
   <class>xIndicatorButton</class>
   <extends>QPushButton</extends>
   <header>xindicatorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../xPad.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>BtnY02</sender>
   <signal>clicked()</signal>
   <receiver>FormWallpaper</receiver>
   <slot>btnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>748</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnY03</sender>
   <signal>clicked()</signal>
   <receiver>FormWallpaper</receiver>
   <slot>btnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>819</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnY04</sender>
   <signal>clicked()</signal>
   <receiver>FormWallpaper</receiver>
   <slot>btnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>890</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnY05</sender>
   <signal>clicked()</signal>
   <receiver>FormWallpaper</receiver>
   <slot>btnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>748</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnY06</sender>
   <signal>clicked()</signal>
   <receiver>FormWallpaper</receiver>
   <slot>btnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>819</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BtnY07</sender>
   <signal>clicked()</signal>
   <receiver>FormWallpaper</receiver>
   <slot>btnclick()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>474</x>
     <y>890</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>btnclick()</slot>
 </slots>
</ui>
